configure.ac

```diff
@@ -406,8 +406,12 @@ if test "${enable_minidriver}" = "yes"; then
 	AC_CHECK_HEADER(
 		[cardmod.h],
 		,
-		[AC_MSG_ERROR([cardmod.h is not found and required for minidriver])]
-	)
+		[AC_MSG_ERROR([cardmod.h from CNG is required for minidriver])],
+		[#if defined(__MINGW32__)
```

Comment:
This check does not influence config.h (besides it was already there before this commit).
It was failing on mingw, however because, as far as I remember, GCC didn't understand some annotations
used by cardmod.h header.